R/functions_util.R

```diff
@@ -11,12 +11,12 @@ first_n_elements_to_string = function(x, n=5, sep=",") {
 }
 
 # Report session info in a table
-scrnaseq_session_info = function() {
+scrnaseq_session_info = function(path_to_git) {
   
   out=matrix(NA, nrow=0, ncol=2)
   colnames(out) = c("Name", "Version")
   
-  repo = system("git log --format='%H' -n 1", intern=TRUE)
+  repo = system(paste0("git --git-dir ", path_to_git, "/.git log --format='%H' -n 1"), intern=TRUE)
```

Comment:
What if git is not installed? Can we have a fallback solution?

Reply:
Mmh, yes sounds good. Idea how? Date of the file scrnaseq.Rmd?

Reply:
Done :-)